web/src/pages/Route/locales/tr-TR.ts

```diff
@@ -183,18 +184,34 @@ export default {
   'page.route.fields.service_id.without-upstream':
     'Hizmeti bağlamazsanız, Yukarı Akışı ayarlamanız gerekir (Adım 2)',
   'page.route.advanced-match.tooltip':
-  'İstek üstbilgileri, istek parametreleri ve tanımlama bilgileri aracılığıyla rota eşleştirmeyi destekler ve gri tonlamalı yayınlama ve mavi-yeşil test gibi senaryolara uygulanabilir.',
+    'İstek üstbilgileri, istek parametreleri ve tanımlama bilgileri aracılığıyla rota eşleştirmeyi destekler ve gri tonlamalı yayınlama ve mavi-yeşil test gibi senaryolara uygulanabilir.',
   'page.route.advanced-match.message': 'İpuçları',
-  'page.route.advanced-match.tips.requestParameter': 'İstek Parametresi：İstek URLsinin sorgulanması',
+  'page.route.advanced-match.tips.requestParameter':
+    'İstek Parametresi：İstek URLsinin sorgulanması',
   'page.route.advanced-match.tips.postRequestParameter':
-  'POST İstek Parametresi：Yalnızca x-www-form-urlencoding formunu destekler',
+    'POST İstek Parametresi：Yalnızca x-www-form-urlencoding formunu destekler',
   'page.route.advanced-match.tips.builtinParameter':
     'Yerleşik Parametre: Nginx dahili parametreleri destekler',
 
   'page.route.fields.custom.redirectOption.tooltip': 'Bu yönlendirme eklentisi ile ilgilidir',
-  'page.route.fields.service_id.tooltip': 'Yapılandırmalarını yeniden kullanmak için Hizmet nesnesini bağlayın.',
+  'page.route.fields.service_id.tooltip':
+    'Yapılandırmalarını yeniden kullanmak için Hizmet nesnesini bağlayın.',
 
   'page.route.fields.vars.invalid': 'Lütfen gelişmiş eşleşme koşulu yapılandırmasını kontrol edin',
   'page.route.fields.vars.in.invalid':
     'IN operatörünü kullanırken parametre değerlerini dizi formatında girin.',
+
+  'page.route.data_loader.import': 'Import',
```

Comment:
why use English here, will it update in the future?

Reply:
@LiteSun 

I don't know any Turkish and I can only translate using a translation tool that may not convey the meaning correctly (I had a lot of problems with the English=>Turkish=>English back-and-forth translation) and I think it's better to use English for now and wait for a later translator to do the translation.

BTW, later I will try to import these into crowdin to simplify the internationalization process.

Reply:
Hi @anldrms, Can you help review these translations?

Reply:
@Baoyuantop @anldrms  If it would help to add a translation of this section, a new PR could be created to do this. ☺️

Reply:
I'll glad to help for that.

Reply:
https://github.com/apache/apisix-dashboard/pull/2487